dart/lib/component/settings_component/settings_component.html

```diff
@@ -158,5 +158,40 @@
         </div>
     </div>
   </tab>
+  <tab heading="Auditor Settings">
+  <br />
+    <div class="col-lg-12">
+        <div class="panel panel-info" ng-switch="isLoaded">
+            <div class="panel-heading">Auditor Settings <span class="badge pull-right">{{auditorlist.length}}</span></div>
+            <div class="panel-body" ng-switch-when="false">
+              <p>Loading . . .</p>
+            </div>
+            <div class="panel-body" ng-switch-when="true">
+                <table class="table table-striped">
+                        <tr>
+                            <th>Issue</th>
+                            <th>Account</th>
+                            <th>Technology</th>
+                            <th>Number of Issues</th>
+                            <th>Disabled?</th>
+                            <th>Enable/Disable</th>
+                        </tr>
+                        <tr ng-repeat="auditoritem in auditorlist">
+                            <td>{{auditoritem.issue}}</td>
+                            <td>{{auditoritem.account}}</td>
+                            <td>{{auditoritem.technology}}</td>
+                            <td><a href="#/issues/-/{{auditoritem.technology}}/{{auditoritem.account}}/-/True/{{auditoritem.issue}}/1/25">{{auditoritem.count}}</a></td>
```

Comment:
The variables should be URL encoded using this method:

https://github.com/Netflix/security_monkey/blob/master/dart/lib/routing/securitymonkey_router.dart#L105

    // Prepares a param for placement in a URL.
    // Replaces null or empty params with URLNULL.
    // URL encodes param.
    String param_to_url(String param) {
        if (param == null || param.isEmpty || param == "null") {
            param = URLNULL;
        }
        param = Uri.encodeComponent(param);
        return param;
    }